### DISCUSSION TITLE
How to add 'swap' with Marks when using 'shortcut userCustomized'

### DISCUSSION THREAD

**Discussion by adi-prakash-ml** [gdh1995/vimium-c#962] [Q&A] [answered]
I know that it is easy to swap the case letter for using local and global marks like this:
```
map ` Marks.activate swap
map m Marks.activateCreateMode swap
```
However, I am using Marks via a shortcut `userCustomized` command (as I want to have these available in all pages like Google Docs, where I have disabled Vimium keybindings. 

I have the following, and it works normally.

```
shortcut userCustomized6 command="Marks.activateCreateMode"
shortcut userCustomized7 command="Marks.activate"
```

But when I add swap like this

```
shortcut userCustomized6 command="Marks.activateCreateMode swap"
shortcut userCustomized7 command="Marks.activate swap"
```

It gives an error saying incorrect command entered.
I also tried:

```
shortcut userCustomized6 command="Marks.activateCreateMode" "swap"
shortcut userCustomized7 command="Marks.activate" "swap"
```
This does not give an error but has no effect i.e. local and global marks casing is not switched.

I would very much appreciate any answers or hints.

**Comment by gdh1995** [comment, marked as answer]
Um... `shortcut xxx command="yyy"` is just like `map <some-shortcut> yyy`, so you just need:
```
shortcut userCustomized7 command="Marks.activate" swap
```

All following items are "command parameters", and their syntax is `name=JsonValue`, like `swap`, `swap=false` and `swap="123"`

**Comment by adi-prakash-ml** [comment]
It works -- thank you very much @gdh1995
